File: SDK/PHP-SDK/ApiTubeConfig.php
<?php

class ApiTubeConfig
{
	private $apiKey;
	private $baseUrl = 'https://api.apitube.io/v1/news';

	public function __construct($apiKey)
	{
		$this->apiKey = $apiKey;
	}

	public function getApiKey()
	{
		return $this->apiKey;
	}

	public function getBaseUrl()
	{
		return $this->baseUrl;
	}
}

File: SDK/Java-SDK/ApiTubeConfig.java
// ApiTubeConfig.java

public class ApiTubeConfig {
    private String apiKey;
    private String baseUrl;

    public ApiTubeConfig(String apiKey) {
        this.apiKey = apiKey;
        this.baseUrl = "https://api.apitube.io/v1/news";
    }

    public String getApiKey() {
        return apiKey;
    }

    public String getBaseUrl() {
        return baseUrl;
    }
}
